testing-overhead/src/test/java/io/opentelemetry/containers/PetClinicRestContainer.java

```diff
@@ -54,6 +53,14 @@ public PetClinicRestContainer(Network network, Startable collector, Agent agent,
         .withExposedPorts(PETCLINIC_PORT)
         .withFileSystemBind(namingConventions.localResults(), namingConventions.containerResults())
         .waitingFor(Wait.forHttp("/petclinic/actuator/health").forPort(PETCLINIC_PORT))
+        .withEnv("spring_profiles_active", "hsqldb,spring-data-jpa")
```

Comment:
If you are using postgre, then `hsqldb` profile shouldn't be needed, right?

Reply:
Good catch, yeah, the `hsqldb` should have been replaced with `postgresql` according to the docs. Fixed it.